content/en/docs/pipelines/sdk/python-function-components.md

```diff
@@ -121,7 +127,7 @@ import kfp.dsl as dsl
   description='An example pipeline that performs addition calculations.'
 )
 def add_pipeline(
```

Comment:
`add_pipeline` -> `calc_pipeline` given you changed this below as

`client.create_run_from_pipeline_func(calc_pipeline, arguments=arguments)`

Reply:
I've left the two samples with different pipeline names, and fixed the bug you pointed out. Thanks!